### PR TITLE
DDP-5847 Translate About Us page to Spanish

### PR DESCRIPTION
**Pull request by aglushko-broad**

### CODE REVIEW

File: ddp-workspace/projects/ddp-mbc/src/app/components/about-us/about-us.component.ts [hunk 1]
```diff
@@ -49,11 +49,9 @@ import { ToolkitConfigurationService } from 'toolkit';
               <a [href]="countMeInUrl" class="Link" target="_blank" translate>Toolkit.AboutUs.ContentLink</a>.
             </p>
             <p class="PageContent-text">
-              <span translate>Toolkit.AboutUs.Content.Paragraph2.Pt1</span>
-              <span class="Italic" translate>Toolkit.Common.Organization</span>'s
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-mbc/src/app/components/about-us/about-us.component.ts:53]: Why did you skip `class="Italic"` ?

Comment (aglushko-broad) [reply ddp-workspace/projects/ddp-mbc/src/app/components/about-us/about-us.component.ts:53]: I use \<em\> element in en.json file which makes it look the same

Comment (alextanlan) [reply ddp-workspace/projects/ddp-mbc/src/app/components/about-us/about-us.component.ts:53]: My bad, didn't notice it firstly.

Comment (alextanlan) [reply ddp-workspace/projects/ddp-mbc/src/app/components/about-us/about-us.component.ts:53]: Interesting, that Spanish  version is a bit different: 
> We leverage the power of <em>Count Me In</em>'s resources

vs 

> Usamos los recursos y tecnología genómica del Instituto Broad (Broad Institute)

**Review by alextanlan — APPROVED** [review]

**Review by MocanaAtBroad — APPROVED** [review]

**Pull request merged**